.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Saw your prettier fixes entering the mix. I guess it is time.

Reply:
🤷🏼‍♂️

Reply:
to semi or not to semi? I've been leaving semis in my newer projects as prettier has been taking care of them for me. doesn't matter to me.

Reply:
Honestly no preference for me. The projects/languages I've been spending time with have them, so I've kinda grown to expect it. And it's default behavior for prettier 🤷‍♂ 

Entirely up to you.